web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -149,4 +192,36 @@ export default {
   border-right: 4px solid transparent;
   border-top: 4px solid var(--font-color-main);
 }
+
+.search-container {
+  min-width: 200px;
+  width: 50%;
+}
+
+.search-panel {
+  display: flex;
+  padding: 5px;
+  border:  1px solid var(--primary-color);
+  border-radius: 4px;
+  background-color: var(--background-color-high-emphasis);
+}
+
+.search-button {
+  align-self: center;
+}
+
+input.search-field {
+  height: 1.5rem;
+  font-size: 1rem;
+  float: right;
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-shadow: none;
+}
+
+input.search-field:not([type]):focus {
+  border-bottom: 0;
+  box-shadow: none;
+}
```

Comment:
```suggestion
.search-panel {
  display: flex;
}

.search-button {
  align-self: center;
}
```

let's keep existing script server styling and don't introduce new styles, it makes the whole project look as a whole and cleaner :)